src/resources/assets/js/mixins/largoContainer.vue

```diff
@@ -469,6 +520,34 @@ export default {
             return this.updateSortKey(SORT_KEY.ANNOTATION_ID)
                 .then(() => this.sortingDirection = SORT_DIRECTION.DESCENDING);
         },
+        handleOpenTab(tab) {
+            if (tab === "annotations" && !this.fetchedAllAnnotations) {
+                this.fetchAllAnnotations();
+            }
+        },
+        fetchAllAnnotations() {
+            this.startLoading();
+            Promise.all([
+                AnnotationsApi.fetchImageVolumeAnnotations({ id: this.volumeId }),
+                AnnotationsApi.fetchVideoVolumeAnnotations({ id: this.volumeId })
+            ])
+                .then(this.parseResponse)
+                .catch(handleErrorResponse)
```

Comment:
The `handleErrorResponse` can only handle API responses. Otherwise the user will only get a generic "the server did not respond" message although there is an entirely different error.

```suggestion
            Promise.all([
                AnnotationsApi.fetchImageVolumeAnnotations({ id: this.volumeId }).catch(handleErrorResponse),
                AnnotationsApi.fetchVideoVolumeAnnotations({ id: this.volumeId }).catch(handleErrorResponse)
            ])
                .then(this.parseResponse)
```